src/sensors.c

```diff
@@ -96,6 +96,8 @@ static bool update_imu(void);
 // function definitions
 void init_sensors(void)
 {
+  // clear the IMU read error
+  _error_state &= ~(ERROR_IMU_NOT_RESPONDING);
```

Comment:
Right now it looks like the `ERROR_IMU_NOT_RESPONDING` flag doesn't get set anywhere?